app/src/main/java/com/duckduckgo/app/browser/pageloadpixel/PageLoadedPixelEntity.kt

```diff
@@ -25,6 +25,5 @@ class PageLoadedPixelEntity(
     val appVersion: String,
     val elapsedTime: Long,
     val webviewVersion: String,
-    val trackerOptimizationEnabled: Boolean,
```

Comment:
I'd rater mark this field as `@Deprecated` and not use to save one migration (more later)